python/HeaderService/hutils.py

```diff
@@ -431,8 +417,8 @@ def write_header(self, filename):
         t0 = time.time()
         if self.write_mode == 'fits':
             self.write_header_fits(filename)
-        elif self.write_mode == 'string':
-            self.write_header_string(filename)
+        elif self.write_mode == 'yaml' or self.write_mode == 'yml':
```

Comment:
How do you get `yml` option?  Does this mean that mimeType above can be `YAML` and `YML`? These should be normalized to YAML early.

Reply:
Actually you are right, `YML` is redundant at this point as this get normalized at an earlier stage. Will fix it.